.changeset/few-foxes-occur.md

```diff
@@ -0,0 +1,8 @@
+---
+'@openfn/lightning-mock': patch
+'dts-inspector': patch
+'@openfn/ws-worker': patch
+'@openfn/cli': patch
+---
+
+Upgrade vulnerable version of ws.
```

Comment:
@josephjclark This is an **alarming comment** - to serve as a friendly reminder that I bumped `ws` several minor versions and, as a result, you thought that some additional testing may be prudent.